APPLEAPPSTORE.LICENSE.WAIVER

```diff
@@ -0,0 +1,19 @@
+The inclusion of Jamulus in the App Store is the only practical way of allowing its use on Apple mobile devices. Although Apple will assert the GPL as the "Custom EULA” for Jamulus, their T&Cs impose some extra conditions. This act of addition when distributing the software is not allowed under the GPL.
+
+However, Apple's terms apply to those who obtain Jamulus from the App Store. It does not affect the copyright holders' assertion of the GPL, or otherwise affect the Jamulus project:
```

Comment:
We should make clear that this is our point of view/understanding/interpretation of the conflict and don't make it sound like we know better.

Reply:
What do you mean by "know better" exactly?

Reply:
We‘re doing a (legal) interpretation of the conflict which we see as correct while some lawyers may not.

Reply:
If there is anything in the statement that might be incorrect, then we need to fix that. We can't have a waiver that we also imply is not a waiver :-)

Reply:
I think after reading through it again, it's clear. There's still one point. See the comment below.